src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -65,18 +65,24 @@ def _get_resource_group(self, **kwargs):
 
 class FlexibleServerMgmtScenarioTest(ScenarioTest):
 
-    postgres_location = 'southeastasia'
-    mysql_location = 'southeastasia'
-
-    @AllowLargeResponse()
-    @ResourceGroupPreparer(location=postgres_location)
-    def test_postgres_flexible_server_mgmt(self, resource_group):
-        self._test_flexible_server_mgmt('postgres', resource_group)
+    postgres_location = 'eastus'
+    mysql_location = 'westus2'
+    mysql_iops_location = 'southeastasia'
```

Comment:
What is the difference between the two ? I am assuming it is because IOPS is deployed in Stage only. But this needs to change to canary as the location because that is where the daily pipeline will run. Also, hold back from checking this PR in until the feature is rolled out in prod. Else Cx will face failures if they try provisioning with iops

Reply:
>  Also, hold back from checking this PR in until the feature is rolled out in prod. Else Cx will face failures if they try provisioning with iops

@arde0708 Can I mark this pr as draft now? You can bring it back to review stage if the feature is rolled out in prod.